modmail/bot.py

```diff
@@ -19,27 +23,130 @@ class ModmailBot(commands.Bot):
     Has an aiohttp.ClientSession and a ModmailConfig instance.
     """
 
-    main_task: asyncio.Task
     logger: ModmailLogger = logging.getLogger(__name__)
 
     def __init__(self, **kwargs):
         self.config = CONFIG
+        self.start_time: t.Optional[arrow.Arrow] = None  # arrow.utcnow()
+        super().__init__(
+            **kwargs,
+        )
         self.http_session: t.Optional[ClientSession] = None
-        self.start_time = arrow.utcnow()
-        super().__init__(command_prefix=commands.when_mentioned_or(self.config.bot.prefix), **kwargs)
 
-    async def create_session(self) -> None:
-        """Create an aiohttp client session."""
-        self.http_session = ClientSession()
+    @classmethod
+    def create(cls, *args, **kwargs) -> "ModmailBot":
+        """
+        Create a ModmailBot instance.
 
-    async def start(self, *args, **kwargs) -> None:
+        This configures our instance with all of our custom options, without making __init__ unusable.
+        """
+        intents = Intents(
+            guilds=True,
+            messages=True,
+            reactions=True,
+            typing=True,
+            members=True,
+            emojis_and_stickers=True,
+        )
```

Comment:
This should be driven by config somehow, or at least specified somewhere constant and not in here.

Reply:
These are the minimally required intents for our bot.